src/api/json/catalog.json

```diff
@@ -3304,13 +3304,13 @@
       "name": "Gradle Build Cache Node",
       "description": "Gradle Build Cache Node configuration schema",
       "fileMatch": [
-        "*build-cache-node-config-schema.yml",
-        "*build-cache-node-config-schema.yaml"
+        "config.yaml"
```

Comment:
The file name is too generic.
Other yaml file can also be called `config.yaml`

This will create issue like #1751 because using generic file name like `build.yml` and `deploy.yml`

The same issue with using generic folder name like `metadata `https://github.com/SchemaStore/schemastore/issues/1783#issuecomment-894368969

Reply:
Ahhh, ok, I'll change it back 👍 

Next time we create an app, I'll make sure we use a less generic name 😉

Reply:
Pushed 189a1ca